robottelo/ui/locators/common.py

```diff
@@ -26,11 +26,9 @@
     "alert.error": (
         By.XPATH, "//div[contains(@class, 'alert-danger')]"),
     "alert.success_sub_form": (
-        By.XPATH,
-        "//div[@ng-hide]//div[contains(@class, 'alert-success')]"),
+        By.XPATH, "//div[contains(@bst-alert, 'success')]"),
```

Comment:
probably yes, but it will require additional verification and out of scope for current PR